scripts/initialize.js

```diff
@@ -612,7 +612,8 @@ var Cookies = require("js-cookie");
               setTimeout(function() {
 							  thisObj.refreshControls('init');
 						  },100);
-            });
+			});
+			thisObj.onLoaded();
```

Comment:
I think we should check if `onLoaded()` is define before calling it

Reply:
ammended @ktml